server/src/assets/game/script-game.js

```diff
@@ -796,6 +803,11 @@ function initialise() {
 		null                                      // trainId
 	);
 
+	var plattform = getPlatformName();
+
+	allPossibleDestinations = eval ("allPossibleDestinations-" + plattform);
+	signalFlagMap = eval("signalFlagMap-" + plattform)
```

Comment:
Spelling inconsistency: "plattform" (wrong) vs. "platform" (correct). Please check other areas of your work for this too.